src/torch_onnx/_verification.py

```diff
@@ -21,13 +38,30 @@ class VerificationInfo:
     # and checked by the runtime
 
 
+@dataclasses.dataclass
+class SearchResult:
+    graph_module: torch.fx.GraphModule
+    inputs: Sequence[Any]
+
+    @property
+    def graph(self) -> torch.fx.Graph:
+        return self.graph_module.graph
+
+    @graph.setter
+    def graph(self, fx_g: torch.fx.Graph):
+        self.graph_module.graph = fx_g
+
+
 def _compare_tensors(
     expected: torch.Tensor,
     actual: torch.Tensor,
 ) -> tuple[float, float]:
     # Move tensors to the same device
     expected = expected.detach().cpu()
     actual = actual.detach().cpu()
+    if expected.dtype == torch.bool:
```

Comment:
Do we need to compare the dtype to make sure they are equal before we compare their values?

Reply:
In theory no because ort would have complained already. But we could check